Packs/MicrosoftGraphDeviceManagement/Integrations/MicrosoftGraphDeviceManagement/MicrosoftGraphDeviceManagement.yml

```diff
@@ -0,0 +1,821 @@
+      description: The reason for the device's access state in Exchange. Possible values are none, unknown, exchangeGlobalRule, exchangeIndividualRule, exchangeDeviceRule, exchangeUpgrade, exchangeMailboxPolicy, other, compliant, notCompliant, notEnrolled, unknownLocation, mfaRequired, azureADBlockDueToAccessPolicy, compromisedPassword, deviceNotKnownWithManagedApp.
+      type: String
+    - contextPath: MSGraphDeviceManagement.Device.IsSupervised
+      description: Device supervised status
+      type: Boolean
+    - contextPath: MSGraphDeviceManagement.Device.IsEncrypted
+      description: Device encryption status
+      type: Boolean
+    - contextPath: MSGraphDeviceManagement.Device.UserPrincipalName
+      description: Device user principal name
+      type: String
+    - contextPath: MSGraphDeviceManagement.Device.Model
+      description: Model of the device
+      type: String
+    - contextPath: MSGraphDeviceManagement.Device.Manufacturer
+      description: Manufacturer of the device
+      type: String
+    - contextPath: MSGraphDeviceManagement.Device.IMEI
+      description: IMEI of the device
+      type: String
+    - contextPath: MSGraphDeviceManagement.Device.SerialNumber
+      description: Serial number of the device
+      type: String
+    - contextPath: MSGraphDeviceManagement.Device.PhoneNumber
+      description: Phone number of the device
+      type: String
+    - contextPath: MSGraphDeviceManagement.Device.AndroidSecurityPatchLevel
+      description: Android security patch level of the device
+      type: String
+    - contextPath: MSGraphDeviceManagement.Device.ConfigurationManagerClientEnabledFeatures.inventory
+      description: Whether inventory is managed by Intune
+      type: Boolean
+    - contextPath: MSGraphDeviceManagement.Device.ConfigurationManagerClientEnabledFeatures.modernApps
+      description: Whether modern application is managed by Intune
+      type: Boolean
+    - contextPath: MSGraphDeviceManagement.Device.ConfigurationManagerClientEnabledFeatures.resourceAccess
+      description: Whether resource access is managed by Intune
+      type: Boolean
+    - contextPath: MSGraphDeviceManagement.Device.ConfigurationManagerClientEnabledFeatures.deviceConfiguration
+      description: Whether device configuration is managed by Intune
+      type: Boolean
+    - contextPath: MSGraphDeviceManagement.Device.ConfigurationManagerClientEnabledFeatures.compliancePolicy
+      description: Whether compliance policy is managed by Intune
+      type: Boolean
+    - contextPath: MSGraphDeviceManagement.Device.ConfigurationManagerClientEnabledFeatures.windowsUpdateForBusiness
+      description: Whether Windows Update for Business is managed by Intune
+      type: Boolean
+    - contextPath: MSGraphDeviceManagement.Device.WiFiMacAddress
+      description: Wi-Fi MAC
+      type: String
+    - contextPath: MSGraphDeviceManagement.Device.HealthAttestationState
```

Comment:
please remove this, as this is a parent for other object and not an output itself